docs/modeling/index.rst

```diff
@@ -323,6 +323,30 @@ are some complexities involved in correctly matching up the inputs and outputs
 of all models used to build a compound model.  You can learn more details in
 the :doc:`compound-models` documentation.
 
+Astropy models also support convolution through the function
+`~astropy.convolution.convolve_models`, which returns a compound model.
+
+For instance, the convolution of two Gaussian functions is also a Gaussian
+function in which the resulting mean (variance) is the sum of the means
+(variances) of each Gaussian.
```

Comment:
I think this statement is correct, see section 4 here: http://www.tina-vision.net/docs/memos/2003-003.pdf